rdm/init_files/data/62304_2006-05_checklist.yml

```diff
@@ -0,0 +1,229 @@
+description: 'Audit checklist for IEC62304'
```

Comment:
Are these leftover files from the old format?

Reply:
Yes. I will delete.